addon/helpers/bs-form-horiz-input-class.js

```diff
@@ -6,8 +6,14 @@ export default helper(function bsFormHorizInputClass([horizontalLabelGridClass]
   if (isBlank(horizontalLabelGridClass)) {
     return undefined;
   }
-  let parts = horizontalLabelGridClass.split('-');
-  assert('horizontalInputGridClass must match format bootstrap grid column class', parts.length === 3);
-  parts[2] = 12 - parts[2];
-  return parts.join('-');
+
+  let originalClasses = horizontalLabelGridClass.split(' ');
+  let inputClasses = originalClasses.map((originalClass) => {
+    let parts = originalClass.split('-');
+    assert('horizontalInputGridClass must match format bootstrap grid column class', parts.length === 3);
+    parts[2] = 12 - parts[2];
```

Comment:
Disclaimer: Not introduced by this PR.

This assumes that a 12 columns grid is used. That's Bootstrap's default value. But it could be changed by setting `$grid-columns` SASS variable as documented here: https://getbootstrap.com/docs/5.3/layout/grid/#sass-variables

Will verify that it actually breaks and open an issue to track it.